_posts/2018-11-26-resize-text.md

```diff
@@ -86,6 +94,7 @@ To do a more thorough job, you should check the page at each of the zoom steps t
 
 * Images of text won't resize
 * Captions over videos likely won't resize
+* Using the incorrect html lang code will result in no text-zoom
```

Comment:
Please use backticks to indicate that `lang` is a code attribute.

Reply:
Done!